tox.ini

```diff
@@ -0,0 +1,59 @@
+[tox]
+envlist = py37
+isolated_build = true
+skipsdist=True
+
+toxworkdir={toxinidir}/.tox
+
+[testenv]
+deps=
+    -r {toxinidir}/requirements/common.txt
+    -r {toxinidir}/requirements/dev.txt 
+commands_pre = 
+    pip install -r {toxinidir}/requirements/docs.txt
+    apt-get install shellcheck
```

Comment:
This will only work in a Linux system. There's another way. See comment below.

```shell
py37 run-test-pre: commands[2] | apt-get install shellcheck
ERROR: InvocationError for command could not find executable apt-get
```